src/ImageSharp/Metadata/Profiles/Exif/Tags/ExifTag.ByteArray.cs

```diff
@@ -21,6 +21,16 @@ public abstract partial class ExifTag
         /// </summary>
         public static ExifTag<byte[]> XMP => new ExifTag<byte[]>(ExifTagValue.XMP);
 
+        /// <summary>
+        /// Gets the IPTC exif tag.
+        /// </summary>
+        public static ExifTag<byte[]> IPTC => new ExifTag<byte[]>(ExifTagValue.IPTC);
```

Comment:
@IldarKhayrutdinov I have trouble with decoding some images from the tiff test-suite.
It seems that some encoders treat the IPTC datatype as **long[]** instead of **byte[]**.
For [example](http://www.luotain.uku.fi/~kraka/uudet_sivut/data/pressi/kraka_pressi02.tif) or [example2](http://korejtko.net/wp-content/uploads/2008/11/148222981.tif)

There are some [sources](https://www.awaresystems.be/imaging/tiff/tifftags/iptc.html) which claim
treating it as **long** is incorrect. I would totally agree, but some encoders do not.
Take for example `iptc.jpg` from our testimages in `Input\Jpg\baseline` and convert it to tif
with either imagemagick or gimp. Both will treat IPTC as **long** for some reason.

Reply:
I think we can remove this line ` public static ExifTag<byte[]> IPTC => new ExifTag<byte[]>(ExifTagValue.IPTC);` the iptc tag object will be created correctly and needs to add the correct processing of iptc tag when IptcProfile creating.

Reply:
Although, the data must be in byte [] format, even if long. Probably should add workaround when reading (exif reader)

Reply:
i think **byte** is the correct datatype. I will create a workaround for the cases when its long.